exercises/D-logical-operators/exercise2.js

```diff
@@ -5,9 +5,26 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
-
-/* 
+function isNegative(num) {
+  if (num > 0) {
+    return false;
+  } else if (num < 0) {
+    return true;
+  }
+  return 0;
+}
+const isBetween5and10 = (num) => {
+  if (num >= 5 && num <= 10) {
+    return true;
+  }
+  return false;
+};
+const isShortName = (name) => (name.length < 7 ? true : false);
```

Comment:
Nice use of the arrow function here. But can you simplify this even more? Maybe even returning the boolean expression directly?